lib/assessment.md

```diff
@@ -41,7 +56,21 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+    ```CSS
+.big {
+  /* All elements that belong to the class `big`*/
+}
+#contentPane {
+  /* The element matching ID `contentPane` */
+}
+.important, .red {
+  /* Any elements that belong both to the `important` and `red` classes */
+}
 
+p {
```

Comment:
Not quite. This will get all `p` elements, not specifically those whose parent elements are `div`s. We can select those elements specifically using the selector `div > p`.